components/header/README.md

```diff
@@ -0,0 +1,25 @@
+### Rationale
+Currently there are three header variants that can be implemented on an SDC service; Standard, Hero, Thin.
+
+### Usage
+
```

Comment:
I think that we should add something here for the implementation of the header about the logo and the addition of a link. For example:
> In use, the logo should be linked to the contextual root of the service if a link is appropriate. For example in SDC the logo would link to https://surveys.ons.gov.uk for respondents.

@nicprice does this accurately describe our previously agreed description for links on the logo? /cc @boxadesign